content/en/docs/languages/java/getting-started.md

```diff
@@ -148,7 +148,8 @@ number of ways, the steps below use environment variables.
    export JAVA_TOOL_OPTIONS="-javaagent:PATH/TO/opentelemetry-javaagent.jar" \
      OTEL_TRACES_EXPORTER=logging \
      OTEL_METRICS_EXPORTER=logging \
-     OTEL_LOGS_EXPORTER=logging
+     OTEL_LOGS_EXPORTER=logging \
+     OTEL_METRIC_EXPORT_INTERVAL=5000
    ```
```

Comment:
I would like to see a sentence here calling out the `OTEL_METRIC_EXPORT_INTERVAL` env var set to 5000 being for local development only, and that the default is 60s.